src/js/components/Pagination/PageControl.js

```diff
@@ -4,11 +4,16 @@ import { ThemeContext } from 'styled-components';
 import { Text } from '../Text';
 import { StyledPaginationButton, StyledContainer } from './StyledPageControl';
 
-export const PageControl = ({ control, separator, ...rest }) => {
+export const PageControl = ({
+  control,
+  separator,
+  size: sizeProp,
+  ...rest
+}) => {
   const theme = useContext(ThemeContext);
 
   return (
-    <StyledContainer as="li">
+    <StyledContainer as="li" sizeProp={sizeProp}>
       {separator ? (
         <Text weight="bold">&#8230;</Text>
```

Comment:
Should the "..." also scale up with the buttons? If so, 
```suggestion
        <Text weight="bold" size={sizeProp}>&#8230;</Text>
```

Reply:
Good call. Done.